include/blogPost.php

```diff
@@ -0,0 +1,48 @@
+<?php
+function getArray(){
+    $post = array(
+        array(
+            'postid'=>0,
+            'title'=> "Getting To Know Me <br/>",
+            'body'=> "<div class = 'aboutMe'>
+                    Below are some more facts about me: <br/> <br/>
+                    I am from NY <br/>
+                     My favorite dessert is ice cream <br/>
+                     My least favorite forrd is chocolate <br/>
+                     I am scared of cats <br/>
+             My favorite Disney movie is mulan <br/><br/><br/>"
+         ),
+         array(
+             'postid'=>1,
+             'title'=> "What I am Studying <br/>",
+             'body'=> "<div class = 'aboutMe'>
+                     I am currently studying CogNeuro<br/><br/><br/>",
+          ));
+          return $post;
+}
+
+function getPost(){
```

Comment:
Another function naming thing: When I read this function name, I think it's saying that the function will return info on a post. Instead, it echos info about all of the posts. If a function echos html, I'd call it something like `printPost` or 'echoPostHtml' or something along those lines. And if it deals with all the posts, I'd make sure to pluralize it, because there's a good chance that in the future you'll have both `getPost` and `getPosts` functions.

A really verbose name for this would be `echoAllPostsHtml` but you can decide for yourself if that's *too* verbose